packages/autorest.python/test/azure/version-tolerant/tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist=py36, py310
+envlist=py36, py310, py311
```

```suggestion
envlist=py37, py311
```
same question: https://github.com/Azure/autorest.python/pull/1552/files#r1012436288

---

ah yeah good change, let me commit the suggestion